the-collab-lab/tcl-73-smart-shopping-list
# Position of sign in/out button

## Description

Position of sign in/out button was moved to top right of screen above the app name.Design choice made by developers. The size of the sign in/out button was also changed. Size was made slightly larger for desktop and a bit larger for mobile device to aid user experience and ability to click button.

## Type of Changes

styling, accessibility 

## Updates

### Before

![Screenshot 2024-04-01 at 8 25 00 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/135680082/46e9de79-a2cd-49cc-91f4-1653f46be4c6)

### After

![Screenshot 2024-04-01 at 8 24 23 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/135680082/50fb718d-e8e0-47c7-be13-5abf58c4326c)

## Testing Steps / QA Criteria

- `git pull`
- `git checkout sign-out-btn-style`
- `npm ci`
- `npm run start`
- from any page in the app observe the new location of the sign in/out button at the top right of the header
- sign in/out functionality still works
